src/main.js

```diff
@@ -45,7 +45,7 @@ let load = Loader({
             expires: key.expires.toJSON(),
           };
         },
-        sentryDNS: async (project) => {
+        sentryDSN: async (project) => {
           return {
             project,
             token:    Statsum.createToken(project, cfg.app.statsum.secret, '25h'),
```

Comment:
Wait, aren't we passing in Statsum.createToken for sentryDSN?

Did we do this backwards?